srcs/python/kungfu/python/__init__.py

```diff
@@ -17,8 +17,6 @@
 def _load_and_init_python_lib():
     _load_clib('libkungfu')
     _python_lib = _load_clib('libkungfu_python')
-    if not os.getenv('KUNGFU_SINGLE_MACHINE_MULTIPROCESS'):
```

Comment:
this will affect existing feature. How about add a new environment variable called `KUNGFU_NO_AUTO_INIT` and `_call_method` based on it?

Reply:
Or just not calling `kungfu_init`?
Follow-up: Do we still need to call `kungfu_finalize`?

Reply:
I think in MindSpore we still prefer to call `kungfu_init` and `kungfu_finalize`. (There is a convenient helper https://github.com/kungfu-ml/kungfu-mindspore/blob/dev/extension/ops/operations/kungfu_comm_ops.py#L81, so you can call them in a simpler way: https://github.com/kungfu-ml/kungfu-mindspore/blob/db67e6f30229d419f0c8c697d09ec730e7b4c112/benchmarks/hello_world.py#L24)

Reply:
actually the test failed due to this commit. I think for now, when running mindspore jobs we should always `export KUNGFU_NO_AUTO_INIT=1`